
string ai_name
string simu_name
geometry_msgs/Point box
geometry_msgs/Point pos
geometry_msgs/Quaternion quat
float64 speed
